morph/cli.py

```diff
@@ -139,14 +168,23 @@ def main():
     p_dump.add_argument('name', metavar='NAME', help='database to dump (all, known, ...)')
     p_dump.add_argument('--freq', action='store_true', help='include frequency as known to MorphMan')
 
+    def add_mizer(parser):
+        parser.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
+                            metavar='NAME',
+                            help='how to split morphemes (%s) (default %s)'
+                                 % (', '.join(MIZERS.keys()), 'mecab'))
+
     p_count = subparsers.add_parser('count', help='count morphemes in a corpus',
                         description='Count all morphemes in the given files and emit a frequency table.')
     p_count.set_defaults(action=cmd_count)
     p_count.add_argument('files', nargs='*', metavar='FILE', help='input files of text to morphemize')
-    p_count.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
-                         metavar='NAME',
-                         help='how to split morphemes (%s) (default %s)'
-                              % (', '.join(MIZERS.keys()), 'mecab'))
+    add_mizer(p_count)
+
+    p_next = subparsers.add_parser('next', help='find next notes to study from a corpus')
+    p_next.set_defaults(action=cmd_next)
+    p_next.add_argument('prio', metavar='FREQS', help='file of morphemes to study, with frequencies')
+    p_next.add_argument('notes', metavar='NOTES', help='file of newline-terminated notes, each tab-separated fields starting with the text')
```

Comment:
Ah, clearly this needs better documentation! Thanks.

I call this format "MorphMan-style" because it's the same form that MorphMan displays in the "database manager", the one given by `Morpheme.show()` -- four tab-separated fields, comprising (1) the written form; (2, 3) the part of speech, in its first and second levels of specificity; (4) the reading. (All of them referring to a base form -- so as you say in another comment, arguably more a "lexeme" than a "morpheme".)

If I were specifying a format from scratch I'd likely make some different choices, but this was already there so I picked it for consistency. Then again, given that you didn't recognize the format, maybe very few other users would either and there isn't much point in that consistency! We could do something different.

I think my preference would be to merge these commands using this format, though, and we can talk about a better one after -- I think it will be easier that way, and also frees us up to iterate on other tweaks to the commands independently with no rush to nail down the better format. When changing to a new format we could always provide a flag like `--format=tabs` to use the old one, if concerned about compatibility.

In any event the format should be documented. This is the same format emitted by `mm dump` -- perhaps a good way to document it would be to explain in that command's help, and refer briefly to `mm dump --help` from wherever else it comes up?